### DISCUSSION TITLE
How to auto pin citation key

### DISCUSSION THREAD

**Discussion by rriffel** [retorquere/zotero-better-bibtex#3068] [Q&A] [answered]
I use Zotero on Overleaf, and on Zotero, I use Betterbibtex and a custom key format. In order to make Overleaf import the key in the custom format, I have to pin it on Zotero. I'm wondering if there is a way to auto-pin the keys. (For example, I use Zotero connector, which automatically generates my keys, but they are not pinned by default. Is there a way to auto-pin them?)

**Comment by retorquere** [comment, marked as answer]
https://retorque.re/zotero-better-bibtex/installation/preferences/index.html#automatically-pin-citation-key-after

**Comment by rriffel** [comment]
Thanks
